### PR TITLE
docs: adding explanation for podman+selinux

### PR DESCRIPTION
**Pull request by bpetit**

### CODE REVIEW

File: docs_src/tutorials/quickstart.md [hunk 1]
```diff
@@ -12,6 +12,18 @@ Or if you downloaded or built a binary, you'd run:
 
     scaphandre stdout -t 15
 
+## Running scaphandre on centos/redhat (family using SELinux) with podman
```
Comment (nmasse-itix) [inline docs_src/tutorials/quickstart.md:15]: "Running scaphandre on Fedora / CentOS Stream / RHEL (or any distribution using SELinux) with podman" ?

**Comment by nmasse-itix** [comment]
I just put one comment on the title, otherwise it looks good to me !

File: docs_src/tutorials/quickstart.md (hunk 1 above)
Comment (nmasse-itix) [reply docs_src/tutorials/quickstart.md:15]: Or even better: "Running scaphandre with podman on Fedora / CentOS Stream / RHEL (or any distribution using SELinux)"

Comment (bpetit) [reply docs_src/tutorials/quickstart.md]: perfect !

**Comment by bpetit** [comment]
> I just put one comment on the title, otherwise it looks good to me !

Thanks a lot !

**Pull request merged**